docs/UserGuide.md

```diff
@@ -1,11 +1,67 @@
-# User Guide for HomeBoss
+# User Guide 
 
-This comprehensive user guide is your key to a seamless start with our cutting-edge software designed specifically for
-home-based businesses. Our solution is tailored to enhance the efficiency of managing delivery addresses, simplifying
-your operations.
+# About HomeBoss
+
+HomeBoss is a desktop application for **home-based business owners** to **manage their customers and deliveries**. 
+HomeBoss streamlines the process of your operations, optimises your resources, and enhances your overall business 
+experience. 
+
+With our application, you will be able to: 
+* Keep track of the details of your customers and deliveries
+* Search for a particular customer/delivery easily
+* Organise your data in a neat and tidy manner by filtering and sorting to your needs </br>
+</br>
+and much more!
+
+All these are done to improve your efficiency and productivity all while being incredibly simple to use.
+HomeBoss is also **optimised for fast typist with the use of a Command Line Interface (CLI)**, allowing you to 
+execute commands quickly. On top of that, we utilise a *Graphical User Interface (GUI)* to let you view your 
+data in a neater and more organised manner. 
+
+Essentially, with HomeBoss, you can **focus on what matters most: running your business**.
 
 ---
 
+# Table of Contents
+
+* 1\. Introduction 
+  * 1.1 About HomeBoss
+  * 1.2 About this User Guide
+  * 1.3 Navigating the User Guide
+* 2\. Table of Contents 
+* 3\. Quick Start Guide 
+  * 2.1 Installation 
+* 4\. Features
+  * 4.1 User
+    * 4.1.1 [Register](#register)
+    * 4.1.2 [Login](#login)
+    * 4.1.3 [Update details](#update-details)
+    * 4.1.4 [Logout](#logout)
+    * 4.1.5 [Account recovery](#account-recovery)
+    * 4.1.6 [Account deletion](#account-deletion)
+  * 4.2 Customer 
+    * 4.2.1 [Add customer](#add-a-customer)
+    * 4.2.2 [Find customers](#find-customers)
+    * 4.2.3 List customers
+    * 4.2.4 Update customer details
+    * 4.2.5 Delete customer
+  * 4.3 Delivery
+    * 4.3.1 Add delivery
+    * 4.3.2 View all deliveries
+    * 4.3.3 View details of deliveries
+    * 4.3.4 Update details of delivery
+    * 4.3.5 Create a note for a delivery 
+    * 4.3.6 Delete delivery
+  * 4.4 Miscellaneous 
+      * [Help](#help)
+      * [Exit](#exit)
+      * [Clear](#clear)
```

Comment:
Perhaps it would be better to have the numbering to be consistent with the other features?